### ISSUE TITLE
Console output error breaks build on Windows 10

### ISSUE THREAD

**Issue by tbeseda** [facebookexperimental/rome#240]
## Description:

Running build-release on Win 10 Pro, PowerShell 7, Node 12.13.0 throws an error and stops the build

> unknown parse/stringMarkup
> ✖ Unclosed blue tag

![screen shot](https://user-images.githubusercontent.com/15697/78192613-bceda100-7435-11ea-9ba9-084c416d8d01.png)

## Steps To Reproduce:

Using [these instructions](https://romejs.dev/docs/introduction/installation):

1. clone rome
1. run build-release
1. get console output seen above

## Expected Results

build-release to finish successfully

Seems to be an issue with the library that formats and prints to the console during build. Specifically the square brackets _inside_ the `<blue>` tag.

Let me know if I can provide more detail or try something different to solve.

**Comment by Lagily** [comment]
Hey @tbeseda could you check if https://github.com/facebookexperimental/rome/pull/242 would fix your issue?

**Comment by tbeseda** [comment]
@Lagily It does progress further in the build, but I get this error
```
✖ Expected to close pad but found brightBlack
<brightBlack><pad count="7" dir="right">--quiet</brightBlack>  no description found
```
![image](https://user-images.githubusercontent.com/15697/78272872-4ef3b900-74cb-11ea-96d0-640b3b660a36.png)

**Comment by Lagily** [comment]
Can you try again @tbeseda?

**Comment by tbeseda** [comment]
@Lagily I'm able to build and install Rome. Some of the console output is a bit off, but I think that would be a different issue/scope:

![image](https://user-images.githubusercontent.com/15697/78274285-35537100-74cd-11ea-8b3d-2f4510b99657.png)

**Comment by Lagily** [comment]
I see, looks like I broke something with the backslash escaping, I'll have a look

**Comment by Lagily** [comment]
the pull request should now also not break the output in your console

**Comment by YMaster** [comment]
# node ./scripts/build-release ./dist
 Building trunk
ℹ Bundling packages/@romejs/cli/bin/rome.ts
✔ Saved to ~/AppData/Local/Temp/rome-dev
 Executing trunk
⚠ Disk caching has been disabled due to the ROME_CACHE=0 environment variable ⚠

 unknown parse/stringMarkup ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Expected token Word but got Slash

  > 1 │   \\\[MemoryFileSystem] Adding new project folder C:\Users\myUserName\rome\\\\
      │                                                                              ^
    2 │         <red><emphasis>^</emphasis></red>

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

✖ Found 1 problem




what can I do?
OS: windows 10

**Comment by sebmck** [comment]
This, along with other Windows improvements, were fixed in #260.
